runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -847,13 +848,9 @@ private CorfuRuntime(@Nonnull CorfuRuntimeParameters parameters) {
                 this.parameters.getMicroMeterRuntimeConfig();
 
         if (microMeterRuntimeConfig.metricsEnabled) {
-            LoggerContext context =  (LoggerContext) LoggerFactory.getILoggerFactory();
-            registry = Optional.ofNullable(context.exists(microMeterRuntimeConfig.configuredLoggerName))
-                    .map(logger -> MeterRegistryProvider.MeterRegistryInitializer.newInstance(logger,
+            Logger logger = LoggerFactory.getLogger(microMeterRuntimeConfig.configuredLoggerName);
+            registry = Optional.of(MeterRegistryInitializer.newInstance(logger,
                             microMeterRuntimeConfig.loggingInterval, parameters.clientId));
-            if (!registry.isPresent()) {
-                log.warn("Logger was not found to enable metrics.");
-            }
```

Comment:
You can probably check if a logger was defined or not with `Logger::isInfoEnabled`

Reply:
done